superset-frontend/src/explore/components/controls/FilterControl/AdhocFilterControl.jsx

```diff
@@ -270,7 +262,7 @@ class AdhocFilterControl extends React.Component {
   optionsForSelect(props) {
     const options = [
       ...props.columns,
-      ...[...(props.formData.metrics || []), props.formData.metric].map(
+      ...[...(props.selectedMetrics || [])].map(
```

Comment:
This is used for allowing selecting adhoc metrics in adhoc filters, but it has stopping working:
https://github.com/apache/superset/issues/13249

The new `selectedMetrics` will be added by a `superset-ui` PR.